### PR TITLE
Let pagination headers pass throug CORS

### PR DESCRIPTION
**Pull request by yogsototh**
Relates to: https://github.com/threatgrid/iroh/issues/1536

### CODE REVIEW

File: src/ctia/http/server.clj [hunk 1]
```diff
@@ -26,11 +26,17 @@
             (split origins-str #","))))
 
 
+(defn- str->vec-of-keywords
+  "take a string with words separated with commas, returns a vec of keywords"
+  [str]
+  (vec (map keyword (split methods-str #","))))
```
Comment (msprunck) [inline src/ctia/http/server.clj]: You could use `mapv` instead of `(vec (map`

File: resources/ctia-default.properties [hunk 2]
```diff
@@ -13,6 +13,8 @@ ctia.http.enabled=true
 ctia.http.port=3000
 ctia.http.access-control-allow-origin=http://(localhost|127.0.0.1)(:\\d+)?
 ctia.http.access-control-allow-methods=get,post,put,delete
+ctia.http.access-control-allow-headers=server,date,content-type,connection,x-total-hits,x-next,x-sort,etag
```
Comment (msprunck) [inline resources/ctia-default.properties]: CORS are from the client POV. Why we need to add server, date and connection to this list?

Comment (gbuisson) [reply resources/ctia-default.properties]: also add x-previous

**Comment by gbuisson** [comment]
the build fails with:

Exception in thread "main" java.lang.RuntimeException: Unable to resolve symbol: methods-str in this context, compiling:(ctia/http/server.clj:32:21)

likely due to a missing require

**Comment by gbuisson** [comment]
perhaps Expose-Headers is more suitable to our case:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers

**Pull request merged**